lms/envs/devstack_docker.py

```diff
@@ -44,7 +44,7 @@
 })
 
 ENABLE_MKTG_SITE = os.environ.get('ENABLE_MARKETING_SITE', False)
-MARKETING_SITE_ROOT = os.environ.get('MARKETING_SITE_ROOT', 'http://wordpress.edx.devstack.lms:8888/')
+MARKETING_SITE_ROOT = os.environ.get('MARKETING_SITE_ROOT', 'http://wordpress.edx.devstack.lms/')
```

Comment:
This change is unrelated to this PR. Please remove it

Reply:
Removed.